app/controllers/order_items_controller.rb

```diff
@@ -0,0 +1,21 @@
+class OrderItemsController < ApplicationController
+  before_action :find_order, only: [:index]
+
+  def index
+    @order_items = Kaminari.paginate_array(@order.order_items
```

Comment:
Gem này vẫn hỗ trợ active record array mà nhỉ
https://github.com/kaminari/kaminari#the-per-scope

Reply:
![Screenshot from 2021-08-02 17-12-37](https://user-images.githubusercontent.com/71007480/127846089-9886f648-334f-4123-b27b-a6b0c806c27b.png)

Reply:
Nó hiện thế á chị